src/import.js

```diff
@@ -539,6 +547,21 @@ Sk.importMainWithBody = function (name, dumpJS, body, canSuspend) {
     return Sk.importModuleInternal_(name, dumpJS, "__main__", body, canSuspend);
 };
 
+/**
+ * **Run Python Code in Skulpt**
+ *
+ * When you want to hand Skulpt a string corresponding to a Python program this is the function.
```

Comment:
@albertjan, since both @LeszekSwirski and I were confused by this, I think it would help us and all future developers if this comment were improved to more clearly explain the purpose of this function and its relation to the other import functions...  The current comment is misleading, in my opinion.